src/openforms/variables/tests/test_rendering.py

```diff
@@ -59,6 +59,7 @@ def test_rendering_user_defined_vars(self):
                         "type": "textfield",
                         "showInEmail": True,
                         "showInPDF": True,
+                        "autocomplete": "on",
```

Comment:
is this needed?

Reply:
Probably not :) - this was one of the first code lines I wrote in this issue.